File: baekjoon/2738/2738.py
'''
행렬 덧셈 python 풀이
https://doctcoder.tistory.com/entry/%ED%96%89%EB%A0%AC-%EB%8D%A7%EC%85%88-%ED%92%80%EC%9D%B4
'''
n,m=map(int,input().split())
mat=[[0 for i in range(m)]for j in range(n)]
for k in range(2):
	for i in range(n):
		tem=list(map(int,input().split()))
		for j in range(m):
			mat[i][j]+=tem[j]
for i in range(n):
	for j in range(m):
		print(mat[i][j],end=' ')
	print('')

File: baekjoon/2738/2738.cpp
/*
행렬 덧셈 C++ 풀이
https://doctcoder.tistory.com/entry/%ED%96%89%EB%A0%AC-%EB%8D%A7%EC%85%88-%ED%92%80%EC%9D%B4
*/
#include <iostream>

using namespace std;
int main()
{
  int n,m;
	cin>>n>>m;
	int arr[101][101]={0,};
	for(int k=0;k<2;k++)
	{
		for(int i=0;i<n;i++)
		{
			for(int j=0;j<m;j++)
			{
				int inp;
				cin>>inp;
				arr[i][j]+=inp;
			}
		}
	}
	for(int i=0;i<n;i++)
	{
		for(int j=0;j<m;j++)
		{
			cout<<arr[i][j]<<" ";
		}
		cout<<"\n";
	}
}
